pt-BR/mail/chrome/messenger/pbDefaultAccountDialog.dtd

```diff
@@ -1,5 +1,5 @@
-<!ENTITY defaultAccount.title  "Default Account">
-<!ENTITY defaultAccount.intro  "Select your default account:">
+<!ENTITY defaultAccount.title  "Conta padrão">
+<!ENTITY defaultAccount.intro  "Selecione sua conta padrão:">
 
-<!ENTITY defaultAccount.desc   "Postbox Free supports one email account.  Upgrade to Postbox to use multiple accounts.">
+<!ENTITY defaultAccount.desc   "O Postbox Free tem suporte a apenas uma conta de e-mail.  Atualize para o Postbox para usar múltiplas contas.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY defaultAccount.desc   "O Postbox Free tem suporte a apenas uma conta de correio eletrônico.  Atualize para o Postbox para usar múltiplas contas.">